catalog/app/containers/Bucket/requests/packageUpload.ts

```diff
@@ -0,0 +1,258 @@
+import * as R from 'ramda'
+import * as React from 'react'
+
+import { JsonValue } from 'components/JsonEditor/constants'
+import * as APIConnector from 'utils/APIConnector'
+import { makeSchemaDefaultsSetter, JsonSchema } from 'utils/json-schema/json-schema'
```

Comment:
is `/json-schema/json-schema` intentional?

Reply:
`json-schema/json-schema` is a mistake

Also, I'm planning to rename `json-schema` to `jsonSchema`